README.md

```diff
@@ -280,6 +303,7 @@ storm::config_map:
     - class: "com.verisign.storm.metrics.GraphiteMetricsConsumer"
```

Comment:
I'm inclined to agree. However, I'm at a loss for a name that'd cover potential future use cases (e.g. InfluxDB, Grafana, etc.) I suppose the only commonality between all of them is that we're feeding in data using the (admittedly poorly named) [plaintext protocol](http://graphite.readthedocs.org/en/latest/feeding-carbon.html?highlight=protocol#the-plaintext-protocol).

Perhaps something like `PlaintextMetricsConsumer`?